vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/VGSCollect.kt

```diff
@@ -702,7 +704,8 @@ class VGSCollect {
      * Warning: if this option is set to false, it will increase resolving time for possible incidents.
      */
     fun setAnalyticsEnabled(isEnabled: Boolean) {
-        tracker.setAnalyticsEnabled(isEnabled)
+        tracker.isEnabled = isEnabled
+        client.getTemporaryStorage().setCustomHeaders(mapOf(generateAgentHeader(isEnabled)))
```

Comment:
Could we move this line to the CollectActionTracker?
```
    override var isEnabled: Boolean = true
    set(value) { <here>}
```

Reply:
@Endorf Unfortunately no, because CollectActionTracker doesn't have reference to client VgsApiTemporaryStorageImpl, CollectActionTracker has it's own VgsApiTemporaryStorageImpl.

Reply:
Actually he has. You can retrieve local store from HttpClient.
https://github.com/verygoodsecurity/vgs-collect-android/blob/master/vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/api/analityc/CollectActionTracker.kt#L25

Reply:
Yes but this another instance of `VgsApiTemporaryStorageImpl`, but we need `vgs-client` header for proxy requests and not analytics